addons/stock_account/models/account_move.py

```diff
@@ -232,6 +234,9 @@ class AccountMoveLine(models.Model):
     _inherit = 'account.move.line'
 
     stock_valuation_layer_ids = fields.One2many('stock.valuation.layer', 'account_move_line_id', string='Stock Valuation Layer')
+    origin_of_cogs_aml_id = fields.Many2one(
+        "account.move.line", copy=False,
+        help="Technical field used to keep track in the originating line of the anglo-saxon lines.")
```

Comment:
```suggestion
    cogs_origin_id = fields.Many2one(  # technical field used to keep track in the originating line of the anglo-saxon lines
        "account.move.line", copy=False,
    )
```

* `help` is intended for end users, not for devs. Putting tech info in there make the UI heavier
* close the parenthesis on the same indentation level than its opening

Reply:
@william-andre Thanks for the reply.

I have applied your suggestions.

Could please review again.

Best Regards.